pipeline/beam_tables.py

```diff
@@ -265,7 +304,7 @@ def _data_to_load(self,
                     gcs: GCSFileSystem,
                     scan_type: str,
                     incremental_load: bool,
-                    table_name: str,
+                    destination: str,
```

Comment:
I would turn this into two optional kv args `table_name` and `file_destination` and switch on whether those are set instead of switching on the content of the string.

Reply:
done